html/html-and-the-web_personal-website/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Janosch Schilling</title>
+  </head>
+  <body>
+    <header>
+      <h1>Janosch Schilling</h1>
+      <nav></nav>
+    </header>
+    <section>
+      <img src="./assets/IMG_8457.jpg" alt="Janosch Schilling" />
+      <p>Hey this is me Janosch, here you see me in a coding bootcamp</p>
+    </section>
+    <aside>
+      <h3>I like</h3>
+      <ul>
```

Comment:
inside a `ul` must be a `li`

Reply:
Fixed the first two issues. For the last comment I left the placeholders for now but thanks!